LanguageFeatures/Patterns/constant_A03_t03.dart

```diff
@@ -23,117 +24,72 @@
 /// literals explicitly marked const. Likewise with set and map literals versus
 /// map patterns.
 ///
-/// @description Check list, map and set literals with type argument specified
-/// in constant patterns
+/// @description Check list, map and set literals in constant patterns. Test
```

Comment:
I believe set literals are not supported, they'd have to use the `case const (<int>{}):` approach (which is working because `<int>{}` is a constant expression when in a constant context, but the plain `case const <int>{}:` should be an error).